CHANGELOG

```diff
@@ -16,6 +16,7 @@ v3.17 (Month 2020)
   * Remove tag color from issue titles in issue summary
   * Use shared noscript partial
   * Use user model reference for activities instead of user email
+    - Add a default user during welcome setup to satisfy model relations
```

Comment:
this is an "internals" type of change, I don't think it's a user-facing CHANGELOG worthy one, no?